src/foam/core/Fluent.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * @license
+ * Copyright 2021 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.core',
+  name: 'Fluent',
+  documentation: `
+    A domain-specific "language" - a model with chainable methods for
```

Comment:
Add:

For creating "fluent" interfaces. See: https://en.wikipedia.org/wiki/Fluent_interface